### PR TITLE
Update testing libraries

### PR DESCRIPTION
**Pull request by joerghoh**
Update the testing libraries to their latest available versions. In preparation for migration to JUnit 5.

* also update joda-time (version 2.9.1 comes with AEM 6.3 and 6.4)

### CODE REVIEW

**Comment by joerghoh** [comment]
Let's wait until aem-mocks 2.7.2 is released; i had to disable a test because of WTES-48 (https://wcm-io.atlassian.net/browse/WTES-48)

File: bundle/src/test/java/com/adobe/acs/commons/filefetch/impl/FileFetchImplTest.java [hunk 1]
```diff
@@ -76,6 +77,8 @@ protected HttpURLConnection openConnection() throws IOException {
   }
 
   @Test
+  @Ignore
+  // Ignore until https://wcm-io.atlassian.net/browse/WTES-48 is resolved
```
Comment (davidjgonzalez) [inline bundle/src/test/java/com/adobe/acs/commons/filefetch/impl/FileFetchImplTest.java]: @joerghoh 2.7.2 is released now.

https://mvnrepository.com/artifact/io.wcm/io.wcm.testing.aem-mock.junit5

Comment (joerghoh) [reply bundle/src/test/java/com/adobe/acs/commons/filefetch/impl/FileFetchImplTest.java]: thanks, will continue later this week.

**Comment by joerghoh** [comment]
Ready for merging, please review :-)

**Pull request merged**